NBitcoin/Secp256k1/ECPubKey.cs

```diff
@@ -304,6 +305,32 @@ private bool secp256k1_eckey_pubkey_tweak_add(ECMultContext ctx, ref GE key, in
 			return true;
 		}
 
+		/// <summary>
+		/// The original function name is `secp256k1_ec_pubkey_combine`
+		/// </summary>
+		public static bool TryCombine(Context ctx, ECPubKey[] pubkeys, out ECPubKey? combinedPubKey)
+		{
+			if (pubkeys == null)
+				throw new ArgumentNullException(nameof(pubkeys));
+
+			combinedPubKey = null;
+			var n = pubkeys.Count();
+			if (n == 0)
+			{
+				return false;
```

Comment:
I would actually prefer if you don't count at all, like the C code. Less code paths (and you could even keep IEnumerable)

Reply:
I think that's a weird api (how do we assure that actual length is longer than given n?).
What about using `IList` if you think array is too strict.

Reply:
why do you want to ensure the actual length is longer than n in the first place? If it is, then it returns false even if you remove this call.

Reply:
Ah, I see. I first thought you are saying we should pass n as a parameter to the function.